src/main/java/frc/robot/operator_interface/OperatorInterface.java

```diff
@@ -36,4 +36,7 @@ public default Trigger getResetGyroButton() {
   public default Trigger getXStanceButton() {
     return new Trigger(() -> false);
   }
+  public default Trigger getVisionIsEnabledSwitch() {
```

Comment:
You need to implement this method in the DualJoysticksOI class and the SingleHandheldOI class and return a specific trigger (choose a less easily accessed button). Refer to the new Operator Interface doc for details.